Jobs/MM_job.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
+
+#   // INITIAL DESCRIPTION //
+"""Run Singlepoint Calculations"""
+
+#   // MEATDATA //
+__author__ = 'Alina Jansen'
+__date__ = '2024-09-19'
+
+#   // IMPORTS //
+
+#   Imports Of Existing Libraries
+import re
+import os
+import math
+import sqlite3
+import sys
+import subprocess
+import numpy as np
+
+#   Imports From Existing Libraries
+
+#   Imports Of Custom Libraries
+
+#   Imports From Custom Libraries
+from Logging.Logger import Logger
+from Generators._helper import filter_xyzq, _flatten
+from Generators.GeneratorGeometries import read_gmx_structure_header, read_gmx_structure_atoms, read_gmx_box_vectors, write_g96
+
+#   // TODOS & NOTES //
+#   TODO:
+#   - Add logger
+#   NOTE:
+
+#   // CLASS & METHOD DEFINITIONS //
+class MM():
+
+    '''
+    This Class Performs A Singlepoint Calculation
```

Comment:
Not true, is it? I know you said docstrings are not updated, but maybe consider putting in the basics here (or at least remove what's distracting).